packages/front-end/components/FactTables/FactMetricModal.tsx

```diff
@@ -2134,6 +2153,7 @@ export default function FactMetricModal({
                     }
                     includeColumn={true}
                     setDatasource={setDatasource}
+                    includeDistinctDates={true}
```

Comment:
We don't need this now, but this could also be pretty useful for proportion metrics as an aggregate filter.  Percent of users who were active for >3 days.